simple-nextjs-app/README.md

```diff
@@ -0,0 +1,14 @@
+Simple [Next.js](https://nextjs.org/) project that expects POST requests from [PayButton](paybutton.org) and validates their authenticity.
+
+## Getting Started
+
+1. Get you user's public key by logging in to [PayButton](paybutton.org) and acessing the **Account** view.
+2. Change `.env` to include your public key, e.g. `PUBLIC_KEY=302a300506032b6570032100615255850442064c84af43736133257cc0bf57a0da5c92baeb4be4856c43c420`
```

Comment:
What's the pub key for?

Reply:
To verify authenticity of the POST requests received as from the user's account of paybutton.org

Reply:
Let's give them more info on this in the readme and/or code.